stdlib/public/TensorFlow/CompilerRuntime.swift

```diff
@@ -307,10 +307,11 @@ public final class _ExecutionContext {
 
     if case .remote(let grpcAddress) = _RuntimeConfig.session {
       debugLog("Setting up the server def to \(grpcAddress)...")
-      let serverDef = TFE_GetServerDef(grpcAddress, status)
+      let serverDef: UnsafeMutablePointer<TF_Buffer>! = TFE_GetServerDef(
```

Comment:
Move the whole function call to the next line.

Reply:
Done.